From 5e1924a6f79c1b07d9bcb4977b58f94efaaa0301 Mon Sep 17 00:00:00 2001
From: Jeff Layton <jlayton@samba.org>
Date: Wed, 16 Jun 2010 10:50:54 -0400
Subject: [PATCH] mount.cifs: use original device name as-is for mtab

We don't want to alter the device name in any way for the mtab as
/bin/umount depends on the string being identical for user mounts.

Signed-off-by: Jeff Layton <jlayton@samba.org>
---
 mount.cifs.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mount.cifs.c b/mount.cifs.c
index 21ce532..51fc1a8 100644
--- a/mount.cifs.c
+++ b/mount.cifs.c
@@ -1953,7 +1953,7 @@ mount_retry:
 	}
 
 	if (!parsed_info->nomtab)
-		rc = add_mtab(dev_name, mountpoint, parsed_info->flags, fstype);
+		rc = add_mtab(orig_dev, mountpoint, parsed_info->flags, fstype);
 
 mount_exit:
 	if (parsed_info) {
-- 
1.7.4.1

